core/internal/testutils/testutils.go

```diff
@@ -452,16 +451,3 @@ func MustDecodeBase64(s string) (b []byte) {
 	}
 	return
 }
-
-// GetFreePort returns a free port.
-// NOTE: This approach is technically incorrect because the returned port
-// can still be taken by the time the caller attempts to bind to it.
-// Unfortunately, we can't specify zero port in P2P.V2.ListenAddresses at the moment.
-func GetFreePort(t *testing.T) uint16 {
```

Comment:
There may be a case for keeping this around solely to return `uint16`, but I'm not sure it matters much in practice.